hasura.planx.uk/metadata/databases/default/tables/public_analytics_aggregated_mini.yaml

```diff
@@ -0,0 +1,3 @@
+table:
+  name: analytics_aggregated_mini
```

Comment:
nit: `mini` isn't inherently meaningful in this context. 

Is there a better way to describe this? Is it maybe best described as a `_rollup`, `_summary`, or just `analytics_aggregated` if we think this will replace the other view? (appreciate we'll need a temp name if that's the case).